README.md

```diff
@@ -83,6 +94,12 @@ The list of built project files. This option should be relative to `distDir` and
 
 *Default:* `context.distFiles`
 
+##### versionFile
```

Comment:
@alisdair The configuration options above are specific to the `FileHash` generator.  The idea was to have a `Configuration Options` section for each key generator because they all have different options that apply.

Could you please reshuffle this so that `File Hash` has a Configuration Options section with `filePattern`, `distDir` and `distFiles`.  The the `Version Commit` generator has it's own section with the `versionFile`?